lib/svrquery/protocol/titanfall/query.go

```diff
@@ -12,7 +17,10 @@ import (
 var (
 
 	// minLength is the smallest packet we can expect.
-	minLength = 26
+	minLength         = 26
+	tagSize           = 16
+	packetSize        = 1200
+	gcmAdditionalData = []byte{1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16}
```

Comment:
this additionalData isn't meant to be private right? (public repo)

Reply:
In here it's not the encryption key, just some additional padding that has been added. Should be ok